From d01397069e857b2440ff077437006474db2025f1 Mon Sep 17 00:00:00 2001
From: Naresh Solanki <naresh.solanki@9elements.com>
Date: Tue, 31 Oct 2023 12:31:14 +0000
Subject: [PATCH 08/13] configurations: sbp1: Update input label name

PVCCD_HV_CPU* input power & current is actualy measure of DIMM power &
current. Hence update the same accordingly.

TEST=Build for sbp1 & make sure the updated labels are reflected.

Signed-off-by: Naresh Solanki <naresh.solanki@9elements.com>
Change-Id: I80110f785d7179dc23927b3e9ecbdd72379d23ff
---
 configurations/sbp1_baseboard.json | 16 ++++++++++++----
 1 file changed, 12 insertions(+), 4 deletions(-)

diff --git a/configurations/sbp1_baseboard.json b/configurations/sbp1_baseboard.json
index 92ce2e5..3e85131 100644
--- a/configurations/sbp1_baseboard.json
+++ b/configurations/sbp1_baseboard.json
@@ -3573,7 +3573,9 @@
                     "Value": 140.0
                 }
             ],
-            "Type": "TDA38640"
+            "Type": "TDA38640",
+            "iin_Name": "CPU3_DIMM_12V_Current",
+            "pin_Name": "CPU3_DIMM_12V_Power"
         },
         {
             "Address": "0x40",
@@ -3634,7 +3636,9 @@
                     "Value": 140.0
                 }
             ],
-            "Type": "TDA38640"
+            "Type": "TDA38640",
+            "iin_Name": "CPU1_DIMM_12V_Current",
+            "pin_Name": "CPU1_DIMM_12V_Power"
         },
         {
             "Address": "0x40",
@@ -3695,7 +3699,9 @@
                     "Value": 140.0
                 }
             ],
-            "Type": "TDA38640"
+            "Type": "TDA38640",
+            "iin_Name": "CPU2_DIMM_12V_Current",
+            "pin_Name": "CPU2_DIMM_12V_Power"
         },
         {
             "Address": "0x40",
@@ -3756,7 +3762,9 @@
                     "Value": 140.0
                 }
             ],
-            "Type": "TDA38640"
+            "Type": "TDA38640",
+            "iin_Name": "CPU0_DIMM_12V_Current",
+            "pin_Name": "CPU0_DIMM_12V_Power"
         },
         {
             "Address": "0x30",
-- 
2.41.0

